website/content/en/preview/upgrade-guide/_index.md

```diff
@@ -36,38 +38,39 @@ When there is a breaking change we will:
 * Increment the minor version when in major version 0
 * Add a permanent separate file named `migrating_to_vx.y.z.md` to our website (linked at the bottom of this page)
   clearly explaining the breaking change and what needs to be done on the customer side to ensure a safe upgrade
-* Add the sentence “This is a breaking change, please refer to `migrating_to_x.y.z.md` for upgrade instructions” to the
-  top of the release notes and in all our announcements
+* Add the sentence “This is a breaking change, please refer to `migrating_to_x.y.z.md` for upgrade instructions” to the top of the release notes and in all our announcements
 
 ## How Do We Find Incompatibilities
 
 Besides the peer review process for all changes to the code base we also do the followings in order to find
 incompatibilities:
-* (To be implemented) Automated tests to see if installing, uninstalling, upgrading from an older version and downgrading
-to an older version to check the compatibility of the application
-* (To be implemented) Turn the commands in our
-documentation into scripts that we can automatically run to find incompatibilities between the application and
-documentation
+* (To be implemented) Automated tests for installing, uninstalling, upgrading from an older version, and downgrading to an older version to check the compatibility of the application
+* (To be implemented) Turn the commands in our documentation into scripts that we can automatically run to find incompatibilities between the application and documentation
 
 ## Nightly Builds
 
-(To be implemented) Every night we will build and release everything that has been checked into the source code. This
-allows us to detect problems including breaking changes and potential drifts in our external dependencies sooner than we
-otherwise would. It also allows some of advanced users who have their own nightly builds to test the upcoming changes
-before they are released.
+(To be implemented) Every night we will build and release everything that has been checked into the source code.
+This allows us to detect problems including breaking changes and potential drifts in our external dependencies sooner than we otherwise would.
+It also allows some of advanced users who have their own nightly builds to test the upcoming changes before they are released.
```

Comment:
Should we change "allows" to "enables" to not repeat allows in to consecutive sentences?
Also should this be "some of our advanced users" or "some of Karpenter's advanced users?"?